setup.py

```diff
@@ -114,77 +104,42 @@ class Extension(extension.Extension, object):
 extensions = [
     Extension(
         name="minorminer._minorminer",
-        sources=["./minorminer/_minorminer" + ext],
+        sources=["./minorminer/_minorminer.pyx"],
         include_dirs=['', './include/', './include/find_embedding'],
         language='c++',
-        extra_compile_args = 'mm',
+        extra_compile_args=extra_compile_args + extra_compile_args_minorminer,
     ),
     Extension(
         name="minorminer.busclique",
-        sources=["./minorminer/busclique" + ext],
+        sources=["./minorminer/busclique.pyx"],
         include_dirs=['', './include/', '.include/busclique'],
         language='c++',
-        extra_compile_args = 'mm',
+        extra_compile_args=extra_compile_args + extra_compile_args_minorminer,
     ),
     Extension(
         name="minorminer.subgraph",
-        sources=["./minorminer/subgraph" + ext] + glasgow_cc,
-        include_dirs=['', './include', './external', './external/glasgow-subgraph-solver/src'],
+        sources=["./minorminer/subgraph.pyx"] + glasgow_cc,
+        include_dirs=['', './include', './external',
+                      './external/glasgow-subgraph-solver/src'],
         library_dirs=['./include'],
         language='c++',
-        extra_compile_args = 'glasgow',
+        extra_compile_args=extra_compile_args + extra_compile_args_glasgow,
     ),
     Extension(
         name="minorminer._extern.rpack._core",
-        sources=["./minorminer/_extern/rpack/_core" + ext_c, "./minorminer/_extern/rpack/src/rpackcore.c"],
+        sources=["./minorminer/_extern/rpack/_core.pyx",
+                 "./minorminer/_extern/rpack/src/rpackcore.c"],
         include_dirs=["./minorminer/_extern/rpack/include"],
         language='c',
```

Comment:
I also noticed we don't specify any additional args for `rpack`. We should consider adding a few optimization flags, though the default `setuptools` flags should be OK for now.